liferay-lima/liferay-portal
# LPS-201181 Fragment for Video Streaming that loads video.js player

Issue: https://liferay.atlassian.net/browse/LPS-201181 

**How to test it:** 

1. Enable Feature Flag `feature.flag.LPS-200108`
2. Go to Home Page and click on edit icon
3. In Fragments section, select **Video Streaming** form Content Display collection
4. Edit the url and add a video from https://gist.github.com/SeunghoonBaek/f35e0fd3db80bf55c2707cae5d0f7184 (these are examples, as we don't have videos on our CDNs yet)

A video player should be rendered and you can play the video you selected.
